noxfile.py

```diff
@@ -203,10 +203,25 @@ def linkcheck(session: nox.Session):
     session.run(*sphinx_commands, *check_hyperlinks, *session.posargs)
 
 
+MYPY_TROUBLESHOOTING = """
+🦺 To learn more about type hints, check out mypy's cheat sheet at:
```

Comment:
Fortunately this particular emoji shows up nicely in the output of GitHub Actions!  I expect that it'll be useful to draw people's attention to the troubleshooting information.